test/AutoValuePluginTest.java

```diff
@@ -16,7 +17,7 @@
 
     @Override
     protected void setUp() throws Exception {
-        VfsRootAccess.SHOULD_PERFORM_ACCESS_CHECK = false; // TODO: a workaround for v15
+//        VfsRootAccess.SHOULD_PERFORM_ACCESS_CHECK = false; // TODO: a workaround for v15
```

Comment:
My tests wouldn't compile with this line. I wasn't sure what to do with it. Running with intellij community edition 2016.